js-exercises/private-props/privateProps.js

```diff
@@ -0,0 +1,26 @@
+function privateProps(obj, isPrivate) {
+    return new Proxy(obj, {
+        set(target, prop, value) {
+            if(isPrivate(prop)) {
+                throw new TypeError("Can't set property "+prop);
+            }
+            else {
+                target[prop] = value;
+                return true;
+            }
+        },
+        has(target, prop) {
+            if(isPrivate(prop)) {
```

Comment:
Can we remove the if else block with just ` return(!isPrivate(prop)) `. Also in case we use single line operations inside if else block we might omit the extra braces for better legibility.

Reply:
I will consider your suggestion and make changes accordingly!